### ISSUE TITLE
Home Page - Drop Down Arrow Missing Name and Value

### ISSUE THREAD

**Issue by jfc3-dol** [presidential-innovation-fellows/worker-dol-gov#5]
While testing using DHS Trusted Tester process, I found the drop down arrow does not have NAME or VALUE when using inspect tool which is an issue for those using a screen reader.

Section 508 - 22(n)
WCAG 2.0 - 1.2.2 Web: Forms
